action.yml

```diff
@@ -44,3 +54,5 @@ runs:
         SPACE_NAME: ${{ inputs.space_name }}
         FEATURE_CHANNEL_BRANCHES: ${{ inputs.feature_channel_branches }}
         ACTION_PATH: ${{ github.action_path }}
+        CHARTS_DIR_PATH: ${{ inputs.charts_dir_path }}
+        IMAGE_NAME: ${{ inputs.image_name }}
```

Comment:
Image Name should come from env, not as an input.

Reply:
Changed to set as environment variable.